config.json

```diff
@@ -168,6 +168,14 @@
         "prerequisites": ["arrays", "callbacks", "errors", "recursion"],
         "status": "beta"
       },
+      {
+        "slug": "high-score-board",
+        "name": "High Score Board",
+        "uuid": "431d1f13-d280-4808-bbc6-a72e628c15c2",
+        "concepts": ["objects"],
+        "prerequisites": ["for-loops"],
```

Comment:
`strings` and `numbers` might be pre-reqs too but I don't know enough about the JS progression. Just pointing it out for someone else's review.

Reply:
We discussed that we don't add the primitives types as prerequisites further down the concept tree in the JS track if they are not a core part of the current concept.